api/clients/codecs/mock/BlobCodec.go

```diff
@@ -0,0 +1,68 @@
+package mock
```

Comment:
nit: filenames should be snakecase i.e. `blob_codec.go`

Reply:
Fixed [a48afb1](https://github.com/Layr-Labs/eigenda/pull/972/commits/a48afb152a4bf90cc901c2f1ba6bfcd23cf90cca)